backend/test/smoke/conftest.py

```
@@ -0,0 +1,150 @@
+        print("Just modified MYSQL_PORT")
+    print(f"ENV FOR INITIALIZATION IS: {set(os.environ.keys())}")
+
+
+
+def pytest_unconfigure(config):
+    """
+    This hook runs after the entire test session finishes,
+    perfect for cleanup.
+    """
+    print("\nCleaning up environment variables...")
+    CURRENT_ENV_KEYS = set(os.environ.keys())
```

Ruff (0.14.6)

Variable `CURRENT_ENV_KEYS` in function should be lowercase

(N806)